addon/components/form.js

```diff
@@ -21,13 +21,14 @@ import validateView, {validateModel} from '../validator/index'
 import {deemberify, recursiveObjectCreate} from '../utils'
 
 const builtinRenderers = {
-  PropertyChooser: 'frost-bunsen-property-chooser'
+  'PropertyChooser': 'frost-bunsen-property-chooser',
+  'select': 'frost-bunsen-input-select',
+  'multi-select': 'frost-bunsen-input-multi-select'
```

Comment:
agreed, it seemed wrong to force authors to write `SelectComponent` when they should just be able to say `select` or `multi-select`